app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
```

Comment:
This was my fault :(  It should be

belongs_to :user 

plural here:
has_many :users 

It's wrong in the other original models.  Thank you for the catch!!